recipes/02/02_where.md

```diff
@@ -0,0 +1,48 @@
+# 02 Filtering
+
+## Script
+
+```sql
+-- This example will show how to filter server logs 
+-- in real-time using a standard WHERE clause.
+-- The table it uses is backed by the `faker` connector 
+-- (https://github.com/knaufk/flink-faker) which 
+-- generates rows in memory and is convenient for testing
+-- queries. As such, it is an alternative to the built-in 
+-- `datagen` connector for testing or prototyping queries.
```

Comment:
Could we move the explanations to the top-level, instead of having them as comments? I find this distracting.

Reply:
I assume, we would you also do this for the CREATE TABLE example then?

Reply:
Ideally, yes. Makes it look a bit cleaner and easier to follow, in my opinion.

Reply:
Thanks. Done :)